build/pr-netfx-validation.yaml

```diff
@@ -42,7 +42,7 @@ stages:
         displayName: Windows Build
         inputs:
           filename: build.cmd
-          arguments: 'Build incremental' # Run an incremental build
+          arguments: 'Build' #Add ' incremental' to build incremental-ly
```

Comment:
Instead of using comments, please just remove it given it's in the scripts how to use it + source control anyway. Otherwise we will never get rid of it